source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.cc

```diff
@@ -297,7 +297,7 @@ class SPIFFEValidatorFactory : public CertValidatorFactory {
     return std::make_unique<SPIFFEValidator>(config, stats, time_source);
   }
 
-  absl::string_view name() override { return "envoy.tls.cert_validator.spiffe"; }
+  std::string name() const override { return "envoy.tls.cert_validator.spiffe"; }
```

Comment:
would do the vice versa, making all name string_view as they are just constants, was it too many places to change?

Reply:
Yeah, too much refactor, this is in `envoy/config/typed_config.h` so it's every other extension.